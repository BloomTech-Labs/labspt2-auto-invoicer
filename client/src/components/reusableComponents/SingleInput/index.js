import React from "react";
import styled from "styled-components";

const StyledSingleInput = styled.input`
  @media (max-width: 500px) {
    height: 40px;
    width: 90%;
  }
`;

const SingleInput = props => {
  return (
    <div>
      <label className="form-label">{props.title}</label>
      <StyledSingleInput
        //className="form-input"
        name={props.name}
        type={props.inputType}
        value={props.content}
        onKeyUp={props.onKeyUp}
        onChange={props.controlFunc}
        placeholder={props.placeholder}
      />
    </div>
  );
};

export default SingleInput;
