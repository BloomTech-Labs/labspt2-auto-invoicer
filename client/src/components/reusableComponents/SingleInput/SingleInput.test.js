import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import SingleInput from "../SingleInput";

describe("<SingleInput />", () => {
  it("renders SingleInput", () => {
    const { asFragment } = render(<SingleInput />);
    expect(asFragment()).toMatchSnapshot();
  });
});
