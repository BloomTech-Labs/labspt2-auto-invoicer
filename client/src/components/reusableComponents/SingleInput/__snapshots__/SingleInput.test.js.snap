// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingleInput /> renders SingleInput 1`] = `
<DocumentFragment>
  <div>
    <label
      class="form-label"
    />
    <input
      class="sc-bdVaJa bYvEDq"
      value=""
    />
  </div>
</DocumentFragment>
`;
